llama_stack/providers/remote/inference/groq/groq.py

```diff
@@ -26,6 +26,7 @@
     Message,
     ResponseFormat,
     ToolChoice,
+    ToolConfig,
```

Comment:
I am curious - how was this not caught by pre-commit checks.

Reply:
🤷🏻 but pytest catches it

Reply:
@terrytangyuan I don't think we have pre-commit checks regarding typing or missing symbols etc. There is no mypy for example.